src/main/java/com/gradle/enterprise/api/BuildCacheBuildProcessor.java

```diff
@@ -1,14 +1,7 @@
 package com.gradle.enterprise.api;
 
 import com.gradle.enterprise.api.client.ApiException;
-import com.gradle.enterprise.api.model.Build;
-import com.gradle.enterprise.api.model.BuildQuery;
-import com.gradle.enterprise.api.model.GradleAttributes;
-import com.gradle.enterprise.api.model.GradleBuildCachePerformance;
-import com.gradle.enterprise.api.model.GradleBuildCachePerformanceTaskExecutionEntry;
-import com.gradle.enterprise.api.model.MavenAttributes;
-import com.gradle.enterprise.api.model.MavenBuildCachePerformance;
-import com.gradle.enterprise.api.model.MavenBuildCachePerformanceGoalExecutionEntry;
+import com.gradle.enterprise.api.model.*;
```

Comment:
I think the convention is to not use star imports

Reply:
Done